unittests/resources/checks/hellocheck_base.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class HelloBaseTest(rfm.RunOnlyRegressionTest, base_test=True):
```

Comment:
Since we only need to check that the prefix is correctly set, I would go for a minimal test:

```python
class TemplateTest(rfm.RegressionTest, template=True):
    pass
```

Reply:
And I would put that in a separate file under `unittests/resources/checks/`